ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -504,6 +504,9 @@ gcp)
     if [ -f "${SHARED_DIR}/gcp_min_permissions.json" ]; then
       echo "$(date -u --rfc-3339=seconds) - Using the IAM service account for the minimum permissions testing on GCP..."
       export GOOGLE_CLOUD_KEYFILE_JSON="${SHARED_DIR}/gcp_min_permissions.json"
+    elif [ -f "${SHARED_DIR}/gcp_user_tags.json" ]; then
```

Comment:
Why we need this dedicated keyfile to run the testing?

Reply:
In order to use organizational level user tags, the IAM service-account should have organizational level Tag User permissions. So we use a pre-configured IAM service-account, i.e. "ipi-xpn-minpt-permissions@openshift-qe.iam.gserviceaccount.com", for such testing.

Reply:
If that, why not directly use `ipi-xpn-minpt-permissions`?

Reply:
I meat, is there an existing key file from `ipi-xpn-minpt-permissions` IAM account.